tools/bazel

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Load the environment variables from the .env file
-source .env
+#source .env
```

Comment:
How do we pass the API key on local builds?

Reply:
Mistake on my part, this should not be commented out. Also we should not write to remote cache from local machines.